HelmetCamera/HelmetCamera.cs

```diff
@@ -92,7 +93,17 @@ public void Start()
             }
 
             // Checking what scene is loaded. We dont need CameraMod in MainMenu, InitScene, InitSceneLaunchOptions
-            bool sceneflag = (SceneManager.GetActiveScene().name != "MainMenu") && (SceneManager.GetActiveScene().name != "InitScene") && (SceneManager.GetActiveScene().name != "InitSceneLaunchOptions");
+            bool sceneflag = true;
+
+            switch (SceneManager.GetActiveScene().name)
+            {
+                case "MainMenu":
+                case "InitScene":
+                case "InitSceneLaunchOptions":
+                    sceneflag = false;
+                break;
+            }
+            

```

Comment:
you can use switch expression which would be even cleaner:

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/switch-expression